_dev/back/index.js

```diff
@@ -121,6 +121,31 @@ $(window).ready(function () {
     }
   });
 
+  // Tab Content : Delete
+  $(document).on('click', '.psre-delete', function () {
+    var idBlock = $(this).data('id');
+    if (confirm(txtConfirmRemoveBlock)) {
```

Comment:
You're also able to return `if (!confirm` to gain one indentation level :)

Reply:
:+1: